static/src/javascripts-legacy/projects/commercial/modules/dfp/prepare-switch-tag.js

```diff
@@ -74,19 +74,25 @@ define([
     function pushAdUnit(dfpDivId, sizeMapping) {
 
         var __switch_zero = window.__switch_zero;
+        var promises = [];
 
         if (__switch_zero) {
             var adUnitIds = findAdUnitIds(sizeMapping.size);
 
             adUnitIds.forEach(function(adUnitId) {
```

Comment:
Yes, however I didn't want to burden the caller with unpacking the return type. Also, there's currently no reason for any other module to care about Switch-specific adunit ids.